IPython/lib/latextools.py

```diff
@@ -15,7 +14,12 @@
 from traitlets.config import get_config
 from traitlets.config.configurable import SingletonConfigurable
 from traitlets import List, Bool, Unicode
-from IPython.utils.py3compat import cast_unicode, cast_unicode_py2 as u
+from IPython.utils.py3compat import cast_unicode, cast_unicode_py2 as u, PY3
+
+if PY3:
```

Comment:
I think it's just a better practice. Checking version numbers directly has often caused bugs because the check ends up being wrong, rather than direct checks for the thing we are using. I know it probably doesn't make a difference in a case like this, but I thin it's a better habit.